electricpy/conversions.py

```diff
@@ -569,4 +569,73 @@ def watts_to_dbmw(watts):
     dbmw = dbw_to_dbmw(dbw)
     return dbmw
 
+
+# Define Voltage to decibel converter
+def voltage_to_db(voltage, ref_voltage):
+    """
+    Voltage to Decibel.
+
+    Given the voltage and reference voltage, this function will evaluate
+    the voltage in the decibel scale.
+
+    Parameters
+    ----------
+    voltage:     float
+                 voltage
+
+    ref_voltage: float
+                 Reference voltage
+    Return
+    ------
+    decibel:    float
+                voltage in the decibel scale
+    """
+
```

Comment:
This blank-line is upsetting the `pydocstyle` automatic CI (basically enforcing "best-practices" in the documentation of the library), if we remove this line (593) then everything should be good-to-go! :+1:

Reply:
Fixed the blank line